_pages/calculator.html

```diff
@@ -114,6 +120,7 @@ <h3>Calculator</h3>
         || ((overall == "") && (exam != "") && (cw == ""))
     ) {
         console.log("At least 2 are empty.")
+        showCalculatorError('At least 2 are empty')
```

Comment:
This one wasn't printed out as an actual error because an error would appear on page load when pre-locking the "overall" field.

Reply:
Ah okay, that makes sense. Should be fixed now 😄